Week-7/Ex-3/p3.js

```diff
@@ -0,0 +1,106 @@
+/* Given a linked list of N nodes.
+The task is to check if the linked list has a loop.
+Linked list can contain self loop.*/
+
+class Node {
+    // constructor
+    constructor(element,next) {
+      this.element = element; 
+      this.next = next;
+    }
+}
+class SingleLinkedList {
+    constructor(value){
+      if(value instanceof Array){
+          this.size = value.length;
+          this.head = value.reverse().reduce((acc,curr) => {
+              if(acc == null){
+                  acc = new Node(curr);
```

Comment:
Don't use curly braces if "If" statement is of one line

Reply:
surely.